src/components/ProductList/ProductList.jsx

```diff
@@ -1,7 +1,72 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useCustomProductList, useExportAllProducts } from './../../hooks/untils';
+import { itemsPerPage } from '../../common/constants';
+import product_list from '../../modules/productList.module.scss'
+import ProductItem from './ProductItem';
 
 
 const ProductList = (props) => {
-	return
-}
+
+	useExportAllProducts()
+	const [isLoading, productList, error] = useCustomProductList()
+
+	//----------------------------------------------------------const
+	const totalItems = productList.length
+	const [currentPage, setCurrentPage] = useState(1)
+	const totalPages = Math.ceil(totalItems / itemsPerPage)
```

Comment:
Don't use magic numbers (some random number without explanation), save this value into a variable or constants

In this example its clear but take this as a thumb rule generally